### DISCUSSION TITLE
How can I add COMMANDLINE_ARGS to my comfy container?

### DISCUSSION THREAD

**Discussion by pokes135** [AbdBarho/stable-diffusion-webui-docker#701] [General]
Can someone point me to how to add COMMANDLINE_ARGS="--force-fp16" to the comfyui container?  If I stop main.py in bash console, it stops the container, so I can't restart the python script with arguments.

Perhaps I can pass arguments with "docker start webui-docker-comfy-1..."  ?

Thank you!

**Comment by Sbaero** [comment]
You can create a file /data/config/comfy/startup.sh and it'll execute all the instructions in order on startup.
Write any commands you wanted to execute then start comfy with any startup flags/commandline args.

Here's my "startup.sh":
python -m pip install sageattention
python -u main.py --listen --port 7860 --use-sage-attention

This works because the author adds whatever you put in this file to the ROOT of execution in entrypoint.sh and it seems to execute before the dockerfile instructions that were hardcoded when you built the image.
